linux下model_check工具编译失败

在  platform/linux目录下执行 ./build_model_check.sh 会报错：
[100%] Linking CXX executable ../../model_check
../../libTNN.so.0.1.0.0: undefined reference to `dlopen'
../../libTNN.so.0.1.0.0: undefined reference to `dlclose'
../../libTNN.so.0.1.0.0: undefined reference to `dlsym'
collect2: error: ld returned 1 exit status
tools/model_check/CMakeFiles/model_check.dir/build.make:114: recipe for target 'model_check' failed
make[2]: *** [model_check] Error 1
CMakeFiles/Makefile2:238: recipe for target 'tools/model_check/CMakeFiles/model_check.dir/all' failed
make[1]: *** [tools/model_check/CMakeFiles/model_check.dir/all] Error 2
Makefile:83: recipe for target 'all' failed
make: *** [all] Error 2

--- a/platforms/linux/CMakeLists.txt
+++ b/platforms/linux/CMakeLists.txt
@@ -5,6 +5,7 @@ include_directories(${CMAKE_CURRENT_SOURCE_DIR}/platforms/android/src)
 
 set(COMMON_SRC ${CMAKE_CURRENT_SOURCE_DIR}/platforms/android/src/test_common.cc)
 
+target_link_libraries(TNN dl)
 
 if(TNN_X86_ENABLE)
     add_library(inference_engine SHARED IMPORTED)

